Error when using material @react-querybuilder/material 8.6.1 with webpack

Hello,

When using material react-querybuilder in a project that uses webpack, the following error is thrown:

```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

Check the render method of `MaterialValueEditor2`.
    at createFiberFromTypeAndProps (react-dom-client.development.js:4259:1)
    at createFiberFromElement (react-dom-client.development.js:4273:1)
    at reconcileChildFibersImpl (react-dom-client.development.js:7879:1)
    at react-dom-client.development.js:8057:1
    at reconcileChildren (react-dom-client.development.js:8621:1)
    at updateFunctionComponent (react-dom-client.development.js:8914:1)
    at beginWork (react-dom-client.development.js:10522:1)
    at runWithFiberInDEV (react-dom-client.development.js:1519:1)
    at performUnitOfWork (react-dom-client.development.js:15132:1)
    at workLoopSync (react-dom-client.development.js:14956:39)

The above error occurred in the <MaterialValueEditor2> component.


```


The error is not thrown in version 7.7.1  

Thank you

Hi, I get errors like the following when using version 8.6.1:

`ERROR in ./node_modules/@react-querybuilder/material/dist/react-querybuilder_material.mjs 1275:0-48
Module not found: Error: Can't resolve '@mui/material/TextField' in 'C:\query_builder\node_modules\@react-querybuilder\material\dist'
Did you mean 'index.js'?
BREAKING CHANGE: The request '@mui/material/TextField' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./src/App.tsx 2:0-68 13:40-60
 @ ./src/index.tsx 4:0-24 8:36-39`

I am attatching a minimal project.

[query_builder.zip](https://github.com/user-attachments/files/20149564/query_builder.zip)

Thanks in advance

--- a/packages/material/src/useMuiComponents.ts
+++ b/packages/material/src/useMuiComponents.ts
@@ -1,22 +1,26 @@
-import CloseIcon from '@mui/icons-material/Close';
-import ContentCopyIcon from '@mui/icons-material/ContentCopy';
-import DragIndicator from '@mui/icons-material/DragIndicator';
-import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
-import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
-import LockIcon from '@mui/icons-material/Lock';
-import LockOpenIcon from '@mui/icons-material/LockOpen';
-import Button from '@mui/material/Button';
-import Checkbox from '@mui/material/Checkbox';
-import FormControl from '@mui/material/FormControl';
-import FormControlLabel from '@mui/material/FormControlLabel';
-import ListSubheader from '@mui/material/ListSubheader';
-import MenuItem from '@mui/material/MenuItem';
-import Radio from '@mui/material/Radio';
-import RadioGroup from '@mui/material/RadioGroup';
-import Select from '@mui/material/Select';
-import Switch from '@mui/material/Switch';
-import TextareaAutosize from '@mui/material/TextareaAutosize';
-import TextField from '@mui/material/TextField';
+import {
+  Close as CloseIcon,
+  ContentCopy as ContentCopyIcon,
+  DragIndicator,
+  KeyboardArrowDown as KeyboardArrowDownIcon,
+  KeyboardArrowUp as KeyboardArrowUpIcon,
+  Lock as LockIcon,
+  LockOpen as LockOpenIcon,
+} from '@mui/icons-material';
+import {
+  Button,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  ListSubheader,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  Select,
+  Switch,
+  TextareaAutosize,
+  TextField,
+} from '@mui/material';
 import { useContext, useMemo } from 'react';
 import { RQBMaterialContext } from './RQBMaterialContext';
 import type { RQBMaterialComponents } from './types';

